app/middleware/logger.js

```diff
@@ -0,0 +1,28 @@
+/*
+  Set up an access logging system using Morgan.
+  Create a write stream using the file system module
+  to write to a log file.
+
+  The function also takes in the app as an argument,
+  which will be passed in from the function in the
+  index.js file contained in the same folder
+*/
+const morgan = require("morgan");
+const fs = require("fs");
+const path = require("path");
+
+module.exports = app => {
+  const folderPath = path.resolve(__dirname, "../logs");
```

Comment:
Can we push the configuration to another directory to make gathering / backing up the logs easier?

Reply:
Where would you like them?

Reply:
Ideally we would be able to set the directory in the configuration, but you can ignore this for now - could just set a symlink on the server.

Reply:
Will leave it alone for the time being then :D